packages/destination-actions/src/destinations/singlestore/index.ts

```diff
@@ -52,23 +53,27 @@ const destination: DestinationDefinition<Settings> = {
     testAuthentication: async (request, { settings }) => {
       const { host, port, username, password, dbName, tableName } = settings
 
-      console.log(JSON.stringify(settings, null, 2))
-
+      // console.log(JSON.stringify(settings, null, 2))
 
       const url = `https://${host}:${port}/api/v2/exec`
       const encodedCredentials = btoa(`${username}:${password}`)
+
       const sql = `
-                CREATE TABLE IF NOT EXISTS \`${tableName}\` (
-                    \`message\` JSON COLLATE utf8_bin,
-                    \`timestamp\` as JSON_EXTRACT_STRING(\`message\`,'timestamp') PERSISTED datetime,
-                    \`event\` AS JSON_EXTRACT_STRING(\`message\`, 'event') PERSISTED VARCHAR(255),
-                    \`messageId\` AS JSON_EXTRACT_STRING(\`message\`, 'messageId') PERSISTED VARCHAR(255),
-                    \`user_id\` AS JSON_EXTRACT_STRING(\`message\`, 'userId') PERSISTED VARCHAR(255),
-                    \`anonymous_id\` AS JSON_EXTRACT_STRING(\`message\`, 'anonymousId') PERSISTED VARCHAR(255),
-                    \`type\` AS JSON_EXTRACT_STRING(\`message\`, 'type') PERSISTED VARCHAR(255),
-                    SHARD KEY ()
-                ) AUTOSTATS_CARDINALITY_MODE=PERIODIC AUTOSTATS_HISTOGRAM_MODE=CREATE SQL_MODE='STRICT_ALL_TABLES';
-            `;
+        CREATE TABLE IF NOT EXISTS ${tableName} (
+          messageId VARCHAR(255) NOT NULL,
+          timestamp DATETIME NOT NULL,
+          type ENUM('track', 'identify', 'page', 'screen', 'group', 'alias') NOT NULL,
```

Comment:
I prefer using VARCHAR(255) here as well. If in the future there will be a new type - we won't need to migrate customers tables